nymvpn-x/src-tauri/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "nymvpn-x"
```

Comment:
I think we should stick to `nym-vpn-x`, since we use "NymVPN" and also that's what everything else is using

Reply:
This app is a standalone client, and as a standalone I don't see any reason to follow naming of other nym apps. As you said the official name of the app is `NymVPN`, so it translates well to -> `nymvpn`-x for the underlying rust crate.

Reply:
I disagree, I think `nym-vpn-x` is better and matches all other crate names have. Also imho the directory should match.  Actually, thinking about it now even better would be `nym-vpn-ui` for the directory, and fits better with the rest of the repo, and puts it in context with the daemon and stuff like that. We can keep the package name and the name that the resulting appimage is generated from to `nymvpn-x` though